# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/bc/use-toolchain-bc.patch.cross
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- bc/bc/Makefile.am.vanilla	2022-01-28 17:10:59.245000000 +0100
+++ bc/bc/Makefile.am	2022-01-28 17:11:42.301000000 +0100
@@ -31,13 +31,10 @@
 
 fbcOBJ = main.o bc.o scan.o execute.o load.o storage.o util.o warranty.o
 
-libmath.h: libmath.b $(fbcOBJ) $(LIBBC)
+libmath.h: libmath.b $(LIBBC)
 	echo '{0}' > libmath.h
-	$(MAKE) global.o
-	$(LINK) -o fbc $(fbcOBJ) global.o $(LIBBC) $(LIBL) $(READLINELIB) $(LIBS)
-	./fbc -c $(srcdir)/libmath.b </dev/null >libmath.h
+	bc -c $(srcdir)/libmath.b </dev/null >libmath.h
 	$(srcdir)/fix-libmath_h
-	rm -f ./fbc ./global.o
 
 sbcOBJ = main.o sbc.o scan.o execute.o global.o load.o storage.o util.o \
          warranty.o
